css/pages/review.scss

```diff
@@ -1,56 +1,3 @@
 // Review
 .page--review {
```

Comment:
Let's delete this page and remove the import from _index.scss since we are no longer using it :)